### PR TITLE
Feat/demo login

### PR DESCRIPTION
**Pull request by guillaumechervetaxa**
### Reference to the issue

`Here you can add link to related issue`

### Description of the issue

`Your description`

### Person(s) for reviewing proposed changes

`You can add @mention here`

# Important

Before creating a pull request run unit tests

```sh
$ npm test

# watch for changes
$ npm test -- --watch

# For a specific file (e.g., in packages/context/__tests__/command.test.js)
$ npm test -- --watch packages/action
```

### CODE REVIEW

**Comment by youf-olivier** [comment]
Any update @guillaume-chervet @guillaumechervet @guillaumechervetaxa (on doit prendre lequels des trois :P)

File: examples/demo/src/Login/Login.validation.rules.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { LOGIN, PASSWORD, MSG_REQUIRED } from './constants';
+
+const rulesRequired = {
```
Comment (makranelhoucine) [inline examples/demo/src/Login/Login.validation.rules.js:3]: I think is better to create one object rules that contains differents rules
for example 

const rules = {
  required: {
        message: MSG_REQUIRED
  },
  maxLength: {
    maxLength: max,
    message: 'Le champ contient trop de caractères',
  },
  number: {
    regex: /^[0-9]*$/,
    message: 'Veuillez saisir un nombre',
  }
};

export const loginFormRules = {
  [LOGIN]: [rules.required],
  [PASSWORD]: [rules.required, rules.maxLength(8), rules.number]
};

File: examples/demo/src/Login/Login.validation.rules.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  required: {
+    message: MSG_REQUIRED,
+  },
+};
+
+const rulesMaxLength = max => ({
+  maxLength: {
+    maxLength: max,
+    message: 'Le champ contient trop de caractères',
```
Comment (makranelhoucine) [inline examples/demo/src/Login/Login.validation.rules.js:12]: move it in constants.js file with MSG_REQUIRED

File: examples/demo/src/Login/Login.validation.rules.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  },
+});
+
+const ruleNumber = {
+  pattern: {
+    regex: /^[0-9]*$/,
+    message: 'Veuillez saisir un nombre',
```
Comment (makranelhoucine) [inline examples/demo/src/Login/Login.validation.rules.js:19]: move it in constants.js file with MSG_REQUIRED

**Comment by arnaudforaison** [comment]
Example has been removed

**Pull request closed without merging**